app/views/admin/manage_users/index.html.erb

```diff
@@ -36,10 +52,16 @@
             <td class="govuk-table__cell"><%= t(user.can_manage_others, scope: 'values') %></td>
             <td class="govuk-table__cell">
               <% unless user.id == current_user_id %>
-                <%= link_to t('.buttons.edit'), edit_admin_manage_user_path(user, return_url: request.original_url), class: 'govuk-link' %>
-                <% unless user.deactivated? %>
-                  <%= link_to t('.buttons.deactivate'), new_admin_manage_user_deactivate_users_path(user), class: 'govuk-link' %>
-                <% end %>
+                <ul class="govuk-summary-list__actions-list">
+                  <% unless user.dormant? %>
```

Comment:
Have we thought about adding pundit or cancancan for these kind of methods - perhaps not complicated enough at this point to worry about it but something to think about.

Reply:
Yes, I don't think we're there yet. I have a draft PR here for restricting access on production which I hope is all we need for now: https://github.com/ministryofjustice/laa-review-criminal-legal-aid/pull/281